README.md

```diff
@@ -1,2 +1,83 @@
-# module-ballerinax-financial.swiftmt
-SWIFT MT message format library
+# Ballerina SWIFT MT Library
+
+## Overview
+
+SWIFT MT (Message Type) messages are a set of standardized financial messages used globally in interbank communication, allowing secure and efficient cross-border transactions. These messages follow a specific structure to ensure consistency in financial data exchange, supporting various operations such as payments, securities trading, and treasury transactions.
+
+The `ballerinax/swiftmt` library provides support for creating, parsing, and manipulating SWIFT MT messages in Ballerina. It facilitates the handling of commonly used MT message types, enabling developers to work with structured financial data in a flexible and programmatic manner. This library is particularly useful for building financial messaging systems that integrate SWIFT MT standards. 
```

Comment:
Wouldn't there be other swift libraries in the future? The name should probably be `swift.mt` rather than `swiftmt`.

Reply:
fixed in `99b8222`